.nengobones.yml

```diff
@@ -150,9 +150,13 @@ docs_conf_py:
   exclude_patterns:
     - tests/test-example.ipynb
   sphinx_options:
-    suppress_warnings: "['image.nonlocal_uri']"
+    suppress_warnings: '["image.nonlocal_uri"]'
   extensions:
     - sphinx_click.ext
   intersphinx_mapping:
     click: https://click.palletsprojects.com/en/7.x
   analytics_id: UA-41658423-2
+
+pre_commit_config_yaml: {}
+
+pyproject_toml: {}
```

Comment:
Should test the `exclude` parameter in both of these.

Reply:
It's used in the configuration notebook. Not sure it's worth adding unnecessary files for testing these very simple options, but I can.

Reply:
Added this in a fixup, see below.